app/assets/javascripts/common/reports.js

```diff
@@ -451,8 +452,9 @@ Reports = function ({ withLtfu, showGoalLines }) {
     return mergeConfig(config, goalLineConfig);
   }
 
-  function lessThan6MonthsSinceFirstPatientRegistered(periodValues) {
-    if (periodValues.length < 6) {
+  function isLessThanDefaultMonthsSinceFirstPatientRegistered(periodValues) {
```

Comment:
Could make a function like so and use that, to save the super long name
```suggestion
  function monthsSinceFirstRegistration(periodValues) {    
      const periodKeysArray = Object.keys(periodValues);
      periodKeysArray.length
  }  
```

Reply:
I like this suggestion as its easy to read.

I did the following

```
if (monthsSinceFirstRegistration(periodValues) < defaultMonthsRequired) {
...
}
```
and I used the new function you suggested to return monthsSinceFirstRegistration